release/smoke_test_1.py

```diff
@@ -0,0 +1,152 @@
+import json
+import os
+import tempfile
+import uuid
+import subprocess
+import sys
+
+
+def start_sd(exe_path):
+    deployment_name = "gravtest%s" % str(uuid.uuid4()).split("-")[4]
+    p = subprocess.Popen("%s launch %s" % (exe_path, deployment_name),
+                         shell=True)
+    rc = p.wait()
+    if rc != 0:
+        raise Exception("Failed to start stardog")
+    return deployment_name
+
+
+def get_status(exe_path, deployment_name):
+    osf, f = tempfile.mkstemp(prefix="graviton_test_env", suffix=".json")
+    p = subprocess.Popen("%s status %s --json-file %s" %
+                         (exe_path, deployment_name, f),
+                         shell=True)
+    rc = p.wait()
+    if rc != 0:
+        raise Exception("Failed to get status")
+    print("Got the status of the deployment.")
+    try:
+        with os.fdopen(osf) as fptr:
+            dict = json.load(fptr)
+        print("Status dict %s" % dict)
+        return str(dict['stardog_url'])
+    finally:
+        os.remove(f)
+
+
+def stop_sd(exe_path, deployment_name):
+    p = subprocess.Popen("%s destroy %s --force" % (exe_path, deployment_name),
+                         shell=True)
+    rc = p.wait()
+    if rc != 0:
+        raise Exception("Failed to destroy the deployment")
```

Comment:
Is there any other info we can dump out about why it failed (or where to look for that info) and/or about what stuff is/was running that needs to be cleaned up manually?